connect/runtime/src/test/java/org/apache/kafka/connect/storage/OffsetUtilsTest.java

```diff
@@ -134,6 +134,16 @@ public void testProcessPartitionKeyValidList() {
         }
     }
 
+    @Test
+    public void testProcessPartitionKeyNullOffset() {
```

Comment:
I think this should be `testProcessPartitionKeyNullPartition` instead?

Reply:
Good catch 😅